### ISSUE TITLE
Pricing issue on place_order_crypto

### ISSUE THREAD

**Issue by davidwmcdevitt** [JestonBlu/RobinHood#132]
I was originally running the following:
```

temp <- get_quote_crypto(RH,"ETC")
      start_quote <- temp$mark_price
      quantity <- input / temp$mark_price
      buy <- place_order_crypto(RH = RH,
                                symbol = "ETC",
                                type = "market",      
                                time_in_force = "gtc", 
                                price = start_quote,  
                                quantity = 1,      
                                side = "buy")       
```

Had an issue where price was being read as "NA," which was solved by converting start_quote to an integer
`
 start_quote <- as.integer(temp$mark_price)`

This allows the order to be placed - but when the confirmation comes in from H the listed price is different than the quoted amount and the rounded version of the quoted amount (price at the time was 68.103, and the order was placed at 67.33).

1) Is there a way to allow the command to read a non-integer for price? 
2) Where is the disconnect between the quoted price and the amount given in the order?

Thanks in advance, happy to dig into this deeper with someone if they'd like.

**Comment by JestonBlu** [comment]
@davidwmcdevitt thanks for the report, i have started looking into this and i have an answer for your first question. 

For the NA issue, the problem is with your precision. `get_quote_crpyto()` returns 5 decimals for ETC, but the API will only accept 3 decimals based on what I just tested. I believe this is instrument specific because you can go above 3 decimals for DOGE. So no need to convert to an integer, just do something like this.

```r
start_quote <- round(temp$mark_price, 3)

buy <- place_order_crypto(RH = RH,
                          symbol = "ETC",
                          type = "market",      
                          time_in_force = "gtc", 
                          price = start_quote,  
                          quantity = 1,      
                          side = "buy")    
```

I still need to investigate the discrepancy, but I was able to reproduce the issue so ill start looking at that shortly. Ill also make a note mention about the precision limits in the docs.

**Comment by JestonBlu** [comment]
@davidwmcdevitt so I have looked into your second question and as far as I can tell it is working correctly.  I see what you mean about the initial order price being different, but it looks like when the order is actually filled the submitted price matches the order record. I experimented with one order and this is what I found.

Ill cut out the irrelevant stuff

```r
place_order_crypto(RH = RH, symbol = "ETC", type = "market", time_in_force = "gtc", 
                   price = 40.02,  quantity = 1, side = "buy")       

$created_at
[1] "2021-06-27 14:58:59 UTC"

$price
[1] 40.02

$quantity
[1] 1

$side
[1] "buy"

$state
[1] "unconfirmed"

$time_in_force
[1] "gtc"

$type
[1] "market"

$updated_at
[1] "2021-06-27 14:58:59 UTC"

```
Then i received an order confirmation email:

```
You have submitted an order to buy $39.62 of Ethereum Classic. 
You can check on the status of this order in your account history. 
We will notify you when this order has been executed.
```

So the email confirmation says the order is lower by 40 cents, but then the final order history on RH shows this.

```
Type
Market Buy

Submitted
Jun 27, 2021

Status
Filled

Entered Amount
$39.62

Filled
Jun 27, 2021, 10:17 AM CDT

Filled Quantity
1.00 Ethereum Classic at $40.01

Total Notional
$40.02

```
So the total notional (which is what I was actually charged) matches the price that was submitted by the command.  I do no know the why the entered amount is 40 cents lower. If anyone knows id be interested in knowing.

One important note is that the price you submit is supposed to represent the maximum you are willing to pay. So theoretically you could put in a price, but you total notional could be lower.

Long story short, i believe its working correctly. And I also think its related to this issue: #129
